### PR TITLE
Adds event post for Google Movie CodeGirl Screening F15; addresses #155

### PR DESCRIPTION
**Pull request by techiechelle**

### CODE REVIEW

File: content/F2015/F15-event-google-movie-code-girl.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: CodeGirl Documentary
+Date: 2015-11-5
+Category: Events
+Tags: social
+Slug: google-movie-code-girl
+Author: Michelle Liauw
+Summary: WiCS and Google presents a documentary screening: CodeGirl
+
+Google and WiCS are inviting you to join us for a screening for CodeGirl: "Join high school-aged girls from around the world as they try to better their community through technology and collaboration in this thrilling, heartfelt documentary."
+
```
Comment (annalorimer) [inline content/F2015/F15-event-google-movie-code-girl.md:13]: Lines need to be 80 characters long

File: content/F2015/F15-event-google-movie-code-girl.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+By 2017, the app market will be valued at $77 Billion. Over 80% of these developers are male. The Technovation Challenge aims to change that by empowering girls worldwide to develop apps for an international competition. From rural Moldova to urban Brazil to suburban Massachusetts, CODEGIRL follows teams who dream of holding their own in the world’s fastest-growing industry. The winning team gets $10K to complete and release their app, but every girl discovers something valuable along the way.
+
```
Comment (annalorimer) [inline content/F2015/F15-event-google-movie-code-girl.md:21]: Same line length thing here.

File: content/F2015/F15-event-google-movie-code-girl.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Watch the trailer here: [link](https://www.youtube.com/watch?v=mrUHMcjwG5A&feature=youtu.be)
```
Comment (annalorimer) [inline content/F2015/F15-event-google-movie-code-girl.md]: Maybe make "here" the link instead? But that's super nitpicky.

File: content/F2015/F15-event-google-movie-code-girl.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Visit [https://www.madewithcode.com/](https://www.madewithcode.com/) and [http://www.codegirlmovie.com/](http://www.codegirlmovie.com/) for more details.
+
+There will be food and Google SWAGGGG!
+
+
+
```
Comment (annalorimer) [inline content/F2015/F15-event-google-movie-code-girl.md]: The extra newlines don't need to be there.

**Comment by annalorimer** [comment]
Looks good Michelle! Thanks for finding all that info, I know it was buried pretty deep.

File: content/F2015/F15-event-google-movie-code-girl.md (hunk 3 above)
Comment (evykassirer) [inline content/F2015/F15-event-google-movie-code-girl.md]: ++ to making here the link

File: content/F2015/F15-event-google-movie-code-girl.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+title: CodeGirl Documentary
+Date: 2015-11-5
+Category: Events
+Tags: social
```
Comment (evykassirer) [inline content/F2015/F15-event-google-movie-code-girl.md]: add industry as a tag too

File: content/F2015/F15-event-google-movie-code-girl.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+By 2017, the app market will be valued at $77 Billion. Over 80% of these 
+developers are male. The Technovation Challenge aims to change that by 
+empowering girls worldwide to develop apps for an international competition. 
+From rural Moldova to urban Brazil to suburban Massachusetts, CODEGIRL follows 
+teams who dream of holding their own in the world’s fastest-growing industry. 
+The winning team gets $10K to complete and release their app, but every girl 
+discovers something valuable along the way.
+
+Watch the trailer here: [here](https://www.youtube.com/watch?v=mrUHMcjwG5A&feature=youtu.be)
```
Comment (evykassirer) [inline content/F2015/F15-event-google-movie-code-girl.md]: `Watch the trailer [here](...)`

File: content/F2015/F15-event-google-movie-code-girl.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+title: CodeGirl Documentary
+Date: 2015-11-5
+Category: Events
+Tags: social, industry
+Slug: google-movie-code-girl
+Author: Michelle Liauw
+Summary: WiCS and Google presents a documentary screening: CodeGirl
```
Comment (annalorimer) [inline content/F2015/F15-event-google-movie-code-girl.md]: replace ':' with 'of'

**Comment by evykassirer** [comment]
looks good! just squash the commits now

**Comment by evykassirer** [comment]
sweet

**Pull request merged**